lib/compile.py

```diff
@@ -197,92 +197,95 @@ def add_schema(table_dict, table):
 
 
 def compile_json(json_file):
-    '''
+    """
     Function to compile JSON script files to python scripts
     The scripts are created with `retriever create_json <script_name` using
     command line
-    '''
+    """
     json_object = json.load(open(json_file + ".json", "r"))
 
-    values = {}
-    values['urls'] = {}
-
-    keys_to_ignore = ["template"]
-
-    for (key, value) in json_object.items():
+    if "retriever" in json_object.keys():
```

Comment:
I think that instead of putting the whole code inside this condition, we could instead print an error and return.